designs/octoplushy/src/head.mjs

```diff
@@ -613,15 +613,16 @@ const options = {
     min: 5,
     max: 500,
     menu: 'style',
+    // eslint-disable-next-line no-unused-vars
     toAbs: (val, { options }, mergedOptions) =>
       ((mergedOptions.sizeConstant * val * 2) / 3.1415) *
-      (options.type === undefined
+      (mergedOptions.type === undefined
         ? 1
-        : options.type == 'octopus'
-        ? 1.7
-        : options.type == 'squid'
-        ? 2
-        : 1),
+        : mergedOptions.type == 'octopus'
```

Comment:
Is there a reason to not use `===` here?

Reply:
Don't think so. Removed